jenkins/attach_agent.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import ConfigParser
+import sys
+
+from jenkinsapi import jenkins
+
+INFO = {
+    'heavy': ('build unittest', 2),
+    'light': ('node e2e', 35),
+    'bespoke': ('bespoke', 10),
+}
+
+
+def info(host, kind):
+    labels, executors = INFO[kind]
+    return {
+        'credential_description': 'Jenkins GCE ssh key',
+        'exclusive': True,
+        'host': host,
+        'java_path': '',
+        'jvm_options': '',
+        'labels': labels,
+        'max_num_retries': 0,
+        'node_description': '',
+        'num_executors': executors,
+        'port': 22,
+        'prefix_start_slave_cmd': '',
+        'remote_fs': '/var/lib/jenkins',
+        'retry_wait_time': 0,
+        'suffix_start_slave_cmd': '',
+    }
+
+
+def create(api, host, config):
+  delete(api, host)
```

Comment:
you used 4 spaces above-- stick to one of them. (PEP8 4-spacing is more universally accepted)